includes/compat/class-wc-stripe-sepa-subs-compat.php

```diff
@@ -482,15 +482,15 @@ public function maybe_render_subscription_payment_method( $payment_method_to_dis
 
 		$stripe_customer->set_id( $stripe_customer_id );
 
-		$sources                   = $stripe_customer->get_sources();
+		$payment_methods  = $stripe_customer->get_payment_methods( 'sepa_debit' );
 		$payment_method_to_display = __( 'N/A', 'woocommerce-gateway-stripe' );
 
-		if ( $sources ) {
-			foreach ( $sources as $source ) {
-				if ( $source->id === $stripe_source_id ) {
-					if ( $source->sepa_debit ) {
+		if ( $payment_methods ) {
+			foreach ( $payment_methods as $payment_method ) {
+				if ( $payment_method->id === $stripe_source_id ) {
+					if ( $payment_method->sepa_debit ) {
```

Comment:
since we are doing `$stripe_customer->get_payment_methods( 'sepa_debit' );` above I think is it not necessary to add this check? all payment methods here must be `sepa_debit` right?